src/raft/raft.rs

```diff
@@ -603,7 +603,9 @@ impl<T: Storage> Raft<T> {
         self.maybe_commit();
     }
 
-    pub fn tick(&mut self) {
+    /// Return true to indicate that some internal states have been changed that
```

Comment:
Strictly speaking, it's possible that the state doesn't change after `tick_heartbeat() -> step MsgCheckQuorum()`.